pkg/components/prometheus-operator/component_test.go

```diff
@@ -27,16 +27,19 @@ func TestEmptyConfig(t *testing.T) {
 	emptyConfig := hcl.EmptyBody()
 	evalContext := hcl.EvalContext{}
 	diagnostics := c.LoadConfig(&emptyConfig, &evalContext)
-	if !diagnostics.HasErrors() {
-		t.Fatalf("Empty config should return error")
+
+	if diagnostics.HasErrors() {
```

Comment:
Why is this needed? What happens if you don't specify grafana admin password? Could you elaborate?

Reply:
The password is automatically generated by the chart, it should be an optional field.